cfme/tests/cloud/test_providers.py

```diff
@@ -70,7 +70,7 @@ def test_add_cancelled_validation(request):
 @pytest.mark.tier(3)
 @pytest.mark.uncollectif(lambda: store.current_appliance.version >= '5.9',
                          reason='no more support for cloud provider discovery')
-def test_password_mismatch_validation(appliance):
+def test_discovery_password_mismatch_validation(appliance):
```

Comment:
**Question**: Doesn't 'cloud' also needed here?

Reply:
@rlbabyuk: I think good to go with discovery only,  as for infra provider discovery not need of password verification. Only cloud provider discovery we need it; specific AWS only. Azure does not support this.